db/migrate/20161014133811_move_content_view_version_description_to_histories.rb

```diff
@@ -1,12 +1,13 @@
 class MoveContentViewVersionDescriptionToHistories < ActiveRecord::Migration
-  class Katello::ContentViewVersion < ApplicationRecord
+  class Katello::FakeContentViewVersion < ApplicationRecord
```

Comment:
To prevent clashes outside of the namespace. Is it likely? Probably not. Is it worth the risk? To me, no.